index.js

```diff
@@ -341,12 +341,25 @@ class GithubScm extends Scm {
                 }
             })
         );
-        const authorLookup = commitLookup.then(commitData =>
-            this.decorateAuthor({
+        const authorLookup = commitLookup.then((commitData) => {
+            const username = (commitData.author) ? commitData.author.login : '_none';
+
+            return this.decorateAuthor({
                 token: config.token,
-                username: commitData.author.login
-            })
-        );
+                username
+            }).catch((err) => {
+                if (err.code !== 404) {
+                    throw err;
+                }
+
+                return {
+                    avatar: 'http://no.ne',
```

Comment:
We should point this to a generic user image hosted on screwdriver instead of a 404.